src/include/duckdb/execution/index/unbound_index.hpp

```diff
@@ -16,15 +16,26 @@ namespace duckdb {
 
 class ColumnDataCollection;
 
+enum class BufferedIndexReplay : uint8_t { IDX_INSERT = 0, IDX_DELETE = 1 };
```

Comment:
Since we already call it `BufferedIndexReplay`, let's just call the entries `INSERT` and `DELETE`?